### ISSUE TITLE
Not working with multiple attempts in a page

### ISSUE THREAD

**Issue by sukanyaagile** [jakiestfu/Youtube-TV#33]
Hi,

We are using this plugin to show our youtube channel videos.It is working fine for the first time. For second time onwards it is not working. Ours is single page app.

We are replacing div content and call the same function the element will not work.

Please consider this.

Thanks.

**Comment by HeathNaylor** [comment]
+1 same issue...

**Comment by HeathNaylor** [comment]
@alexiskattan this is fixed by my pull request. Do you still use this library? It has been inactive for a few years, debating on picking it up as a fork and at least getting some PR's satisfied if there is enough interest.

**Comment by alexiskattan** [comment]
@HeathNaylor Yes, still using it.  I think a fork would be great.  I tried using the edits #43 but it only load the last instance of the player when there are multiple on the page.  I'm using this verion https://github.com/jakiestfu/Youtube-TV/issues/21.  #21 loads the player multiple times but from time to time doesn't load the videos inside the player.  It is intermittent.  If there is any fix I'd be super interested.
